substrate/client/network/src/service.rs

```diff
@@ -90,6 +90,8 @@ use sc_utils::mpsc::{tracing_unbounded, TracingUnboundedReceiver, TracingUnbound
 use sp_runtime::traits::Block as BlockT;
 
 pub use behaviour::{InboundFailure, OutboundFailure, ResponseFailure};
+// Import our custom type 
+pub use crate::request_responses::{CustomOutboundFailure,CustomInboundFailure};
```

Comment:
Why is it pub in `service`? It's already public in `request_responses`, where it belongs.